app/core/src/main/java/com/fsck/k9/K9.kt

```diff
@@ -152,6 +152,9 @@ object K9 : EarlyInit {
     @JvmStatic
     var lockScreenNotificationVisibility = LockScreenNotificationVisibility.MESSAGE_COUNT
 
+    @JvmStatic
```

Comment:
This class is marked as deprecated and `GeneralSettings` should be used instead. Is this still true?

Reply:
Probably not. Moving to `GeneralSettings` would just get rid of the global `K9` object, but keep the central location for all settings.

> At the moment all configuration is derived from one central location. Wouldn't it make sense to move this kind of configuration close to the feature?

Yes. However, we still want to use a central settings storage. We need to come up with a proper architecture for how to do that first.